projects/vtk.cmake

```diff
@@ -10,12 +10,12 @@ if (UNIX)
   endif ()
 endif ()
 
-set(vtk_smp_type "STDThread")
-set(vtk_smp_enable_stdthread OFF)
+set(vtk_smp_type "Sequential")
+set(vtk_smp_enable_sequential OFF)
 if (tbb_enabled)
   set(vtk_smp_type "TBB")
 else ()
-  set(vtk_smp_enable_stdthread ON)
+  set(vtk_smp_enable_sequential ON)
```

Comment:
can you remind me why stdthread is a no go ?

Reply:
Failing compilation on a specific OS (can't remember which one)